### DISCUSSION TITLE
OpenID compliance check is not based on the final specification

### DISCUSSION THREAD

**Discussion by elythh** [rabbitmq/rabbitmq-server#10987] [Other] [answered]
### Describe the bug

Hello 👋🏼 
During the compliance check, RabbitMQ is currently looking for the `end_session_endpoint` argument[1]. However, this endpoint is not part of the final[2] specification but rather a draft[3].

[1] https://github.com/rabbitmq/rabbitmq-server/blob/e610adaa113fc10e26356f981794d40113965bb4/deps/rabbitmq_management/priv/www/js/oidc-oauth/helper.js#L269
[2] https://openid.net/specs/openid-connect-session-1_0.html
[3] https://openid.net/specs/openid-connect-session-1_0-17.html

### Reproduction steps

1. Configure rabbitmq with Auth0 as oauth2 provider
2. Try to log into your app
3. Get prompted with bad compliance

### Expected behavior

This check should not be part of the compliance check.

### Additional context

For anyone else having the same issue with 3.13.x, you can still manually enable that endpoint discovery following [this](https://auth0.com/docs/authenticate/login/logout/log-users-out-of-auth0#configure-rp-initiated-logout) documentation

EDIT: Any reason why the previous one got delete ? https://github.com/rabbitmq/rabbitmq-server/issues/10969

**Comment by michaelklishin** [comment]
@elythh there is a very specific reason why it was deleted: it was moved to a discussed and answered https://github.com/rabbitmq/rabbitmq-server/discussions/10972#discussioncomment-9087339.

**Comment by michaelklishin** [comment, marked as answer]
I don't have anything to add to https://github.com/rabbitmq/rabbitmq-server/discussions/10972#discussioncomment-9087339. Unless this becomes problematic for paying users, there are no plans to change anything besides adding a doc note to the Auth0 example. This is not an issue for other identity providers either.

Our team does not use issues for discussions => this will be locked.

**Comment by michaelklishin** [comment]
FTR, we have encountered a similar complaint from a paying user and here is the current (very brief) change plan: https://github.com/rabbitmq/rabbitmq-server/issues/11067.

**Reply by michaelklishin** [reply]
And https://github.com/rabbitmq/rabbitmq-website/pull/1893 is the docs part.
